### PR TITLE
Read airtable from config

### PR DESCRIPTION
**Pull request by madamc**
Added config details for the airtable datastore in the load_configs.py

the config varies slightly from the issue this is linked to. Instead of an Airtable_connection_info section in the config file, I added a DEFAULT section which has the same data plus a couple of other details. DEFAULT allows easy overriding if desired and removes the need to define default values in the custom sections themselves.

Additionally, I tried to set it so that static config values are specifically called instead of tuple values which work better for the dynamic config settings.

### CODE REVIEW

File: wikitongues/wikitongues/config/indexing.cfg [hunk 1]
```diff
@@ -26,4 +26,28 @@ wikipedia : WikipediaSpider
...
+# fake : true
+
+[DEFAULT]
+# MODIFY
+base_id : {your base id}
+api_key : {your api key}
+# DO NOT MODIFY
+fake : false
+table_name : Languages
+id_column : Identifier
```
Comment (smrohrer) [inline wikitongues/wikitongues/config/indexing.cfg:53]: There shouldn't be default values for these properties. Instead uncomment the settings for these two properties in each of the two table-specific sections

File: wikitongues/wikitongues/config/load_configs.py [hunk 2]
```diff
@@ -43,3 +48,26 @@ def load_configs():
         # nothing overridden. return the default config settings
         print("Using default configuration")
         return default_config
+
+def load_airtable_datastores(config):
```
Comment (smrohrer) [inline wikitongues/wikitongues/config/load_configs.py:52]: Could you make this two separate functions, one for each table

**Review by smrohrer — CHANGES_REQUESTED** [review]

File: wikitongues/wikitongues/config/indexing.cfg (hunk 1 above)
Comment (madamc) [reply wikitongues/wikitongues/config/indexing.cfg:53]: alright, sounds good.

File: wikitongues/wikitongues/config/load_configs.py (hunk 2 above)
Comment (madamc) [reply wikitongues/wikitongues/config/load_configs.py]: yep

**Review by smrohrer — APPROVED** [review]
Nice work! Very satisfying to see language_indexing.py look a lot cleaner :-)

**Comment by smrohrer** [comment]
Looks good, let's get the flake8 issues cleaned up and then this will be ready to merge

**Comment by madamc** [comment]
Cool. I'll knock those out.

**Review by smrohrer — APPROVED** [review]

**Pull request merged**